src/plugins/tools/apipTextToSpeech/plugin.js

```diff
@@ -224,6 +225,7 @@ export default pluginFactory({
         // Handle plugin button click
         this.button.on('click', (e) => {
             e.preventDefault();
+            this.setState('sleep', false);
```

Comment:
Only when clicking on the button? Maybe could it be better to have this when doing the action instead or am I mistaken? (when catching the event)

Reply:
yes, has a reason, fixed